src/info/indexes_info.c

```diff
@@ -31,17 +31,22 @@ TotalIndexesInfo IndexesInfo_TotalInfo() {
     }
     // Lock for read
     pthread_rwlock_rdlock(&sp->rwlock);
-    size_t cur_mem = IndexSpec_TotalMemUsage(sp, 0, 0, 0);
-    info.total_mem += cur_mem;
-    if (info.min_mem > cur_mem) info.min_mem = cur_mem;
-    if (info.max_mem < cur_mem) info.max_mem = cur_mem;
-    info.indexing_time += sp->stats.totalIndexTime;
 
     // Vector indexes stats
     VectorIndexStats vec_info = IndexSpec_GetVectorIndexesStats(sp);
     info.fields_stats.total_vector_idx_mem += vec_info.memory;
     info.fields_stats.total_mark_deleted_vectors += vec_info.marked_deleted;
 
+    // Todo: today, this is a best effort does not account for the entire index memory accurately.
+    // In particular, it does not account for the vector index memory.
```

Comment:
If you prefer not to include the vector index memory directly in `IndexSpec_TotalMemUsage`, I'd suggest refining the comment:
```suggestion
    // Currently, we don't track memory usage at allocation time. 
    // Instead, `IndexSpec_TotalMemUsage` provides an estimate by aggregating the memory usage of specific index components. 
// It does not include the memory used by the vector index, so we need to account for it manually here.
```